Add additional types and note about selecting type (#1729)
timescale/docs: api/refresh_continuous_aggregate.md

## refresh_continuous_aggregate() <tag type="community">Community</tag>

Refresh all buckets of a continuous aggregate in the refresh window given by
`window_start` and `window_end`.

A continuous aggregate materializes aggregates in time buckets (for example,

> min, max, average over 1 day worth of data), as determined by the `time_bucket`

Reviewer: ```suggestion
min, max, average over 1 day worth of data and isdetermined by the `time_bucket`
```

> interval specified when the continuous aggregate was created. Therefore, when

Reviewer: ```suggestion
interval. Therefore, when
```

refreshing the continuous aggregate, only buckets that completely fit within the
refresh window are refreshed. In other words, it is not possible to compute the

> aggregate over, for example, half a bucket. Therefore, any buckets that do not

Reviewer: ```suggestion
aggregate over, for an incomplete bucket. Therefore, any buckets that do not
```
